server/handlers/user_handler.coffee

```diff
@@ -497,8 +497,6 @@ UserHandler = class UserHandler extends Handler
       photoURL = document?.get('photoURL')
       if photoURL
         photoURL = "/file/#{photoURL}"
```

Comment:
Shouldn't we stop using photoURL on client and server, and mark the property deprecated?